.github/scala-steward.conf

```diff
@@ -3,7 +3,7 @@ pullRequests.frequency = "@monthly"
 commits.message = "[7.x] ${artifactName} ${nextVersion} (was ${currentVersion})"
 
 pullRequests.grouping = [
-  { name = "patches", "title" = "Patch updates", "filter" = [{"version" = "patch"}] }
+  { name = "patches", "title" = "[7.x] Patch updates", "filter" = [{"version" = "patch"}] }
```

Comment:
ah yeah, so we had it correct for "normal" pr, but not the grouped patched ones...